packages/gestalt/src/TextArea.tsx

```diff
@@ -215,7 +222,29 @@ const TextAreaWithForwardRef = forwardRef<HTMLTextAreaElement, Props>(function T
     maxHeight: rows * ROW_HEIGHT + INPUT_PADDING_WITH_TAGS,
   } as const;
 
-  return (
+  return isInVRExperiment ? (
```

Comment:
nit: can we return the component and not use an `if,else`

Also, is this an area where can use spread `props` safely?

Since it's an internal component with the exact same props

Feels like it could look cleaner in 2 lines, if there's nothing special happening:
```
if(isInVRExperiement)
`return <VRTextArea  ...props/>`
```

Reply:
I like being explicit with props, I'm not passing all props to all variants. I like destructuring at the very top and setting defaults there.

Reply:
The if/selse is being refactored in the follow up PR where I implement tags